netperf_tool_interfaces/msg/Bytes.msg

```diff
@@ -0,0 +1,3 @@
+uint64  timestamp
```

Comment:
I'm curious why you don't use the `builtin_interfaces/msg/Time` type?

Reply:
It was easier, so I don't need to convert the timestamps in its `sec` and `nsec` components.

Reply:
Is this really the right reason to avoid the Time type? We have facilities to handle the conversions for you. I left a few comments pointing out how.

Reply:
Done, see https://github.com/ros2/ros2cli/pull/720/commits/084b304892a29231a93a5a0f46fa8b9ad349b639